contracts/Escrow.sol

```diff
@@ -333,7 +379,7 @@ contract Escrow {
             _totalStEthInEthLocked + _totalWstEthInEthLocked + _totalWithdrawalNftsAmountLocked;
         rageQuitSupport = (totalRageQuitStEthLocked * 10 ** 18) / stEthTotalSupply;
 
-        uint256 totalStakedEthLocked = totalRageQuitStEthLocked + _totalWithdrawalNftsAmountLocked;
+        uint256 totalStakedEthLocked = totalRageQuitStEthLocked + _totalFinalizedWithdrawalNftsAmountLocked;
         totalSupport = (totalStakedEthLocked * 10 ** 18) / stEthTotalSupply;
```

Comment:
I'd suggest to mention in method name that it returns a percent not a totalSupport in steth